test/test_mixed_KF_UKF/main.cpp

```diff
@@ -55,10 +57,14 @@ class MixedKFUKFSimulation : public GaussianFilter
 };
 
 
-int main()
+int main(int argc, char* argv[])
 {
     std::cout << "Running a UKF filter with KF prediction and UKF correction on a simulated target." << std::endl;
-    std::cout << "Data is logged in the test folder with prefix testKF_UKF." << std::endl;
+
+    const bool write_to_file = (argc > 1 ? std::string(argv[1]) == "ON" : false);
+    if (write_to_file)
+        std::cout << "Data is logged in the test folder with prefix testSIS." << std::endl;
```

Comment:
I think it should be `... with prefix testKF_UKF`.

Reply:
Done!